src/__tests__/App.test.js

```diff
@@ -19,6 +31,48 @@ describe('App', () => {
     await fireEvent.click(queryByText(/pomodoro/i));
     expect(tomatoButton.textContent).toMatch(/25:00/);
   });
+  it('pomodoro timer shows 20:00 when 5 minutes have passed', async () => {
+    jest.useFakeTimers();
+
+    render(App);
+
+    const tomatoBtn = queryByTestId('tomato-button');
+    const fiveMinutesInMs = 5 * 60 * 1000;
+
+    expect(tomatoBtn.textContent).toMatch(/25:00/);
+
+    await fireEvent.click(tomatoBtn);
+    await jest.advanceTimersByTime(fiveMinutesInMs);
+    expect(tomatoBtn.textContent).toMatch(/20:00/);
+
+    jest.runOnlyPendingTimers();
+    jest.useRealTimers();
+  });
+
+  it('pomodoro timer shows 0 seconds left when 25 minutes have passed', async () => {
```

Comment:
I like your audio test, let's rename this to call out that's what we're testing!
```suggestion
  it('plays audio when timer reaches 0:00', async () => {
```